website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "0.60",
```

Comment:
Just change this to [0.60-RC](https://github.com/facebook/react-native-website/blob/master/website/versions.json#L2) and I think it will be fine.

Reply:
Done!